### PR TITLE
feat: implement flush()

### PR DESCRIPTION
**Pull request by dunglas**
Closes #11.

To test it: `curl --no-buffer http://localhost/flush.php`

### CODE REVIEW

File: frankenphp.c [hunk 1]
```diff
@@ -439,6 +439,15 @@ static int frankenphp_send_headers(sapi_headers_struct *sapi_headers)
 	return SAPI_HEADER_SENT_SUCCESSFULLY;
 }
 
+static void frankenphp_sapi_flush(void *server_context)
+{
+	frankenphp_server_context *ctx = (frankenphp_server_context *) server_context;
+
+	if (!ctx || ctx->current_request == 0) return;
```
Comment (withinboredom) [inline frankenphp.c:446]: I wonder if we should call `php_handle_aborted_connection()` if `ctx` is set?

**Review by withinboredom — APPROVED** [review]
Works as advertised

File: frankenphp.c (hunk 1 above)
Comment (withinboredom) [reply frankenphp.c:446]: This doesn't seem to have any effect.

File: frankenphp_test.go [hunk 2]
```diff
@@ -436,6 +438,40 @@ func testEarlyHints(t *testing.T, opts *testOptions) {
...
+func TestFlush_module(t *testing.T) { testFlush(t, &testOptions{}) }
+func TestFlush_worker(t *testing.T) {
+	testFlush(t, &testOptions{workerScript: "flush.php"})
+}
+func testFlush(t *testing.T, opts *testOptions) {
+	runTest(t, func(handler func(http.ResponseWriter, *http.Request), _ *httptest.Server, i int) {
+		var j int
+
+		req := httptest.NewRequest("GET", fmt.Sprintf("http://example.com/flush.php?i=%d", i), nil)
+		w := &streamResponseRecorder{httptest.NewRecorder(), func(buf []byte) {
```
Comment (withinboredom) [inline frankenphp_test.go:461]: Clever! I assume the `i` variable is to ensure the response is sent to the correct client?

Comment (dunglas) [reply frankenphp_test.go:461]: exactly

**Pull request merged**